apps/web/components/sections/project/success-gallery.tsx

```
@@ -0,0 +1,36 @@
+    <div className="bg-white rounded-lg p-6 shadow-md border border-gray-200">
+      <div className="flex items-center justify-between mb-4">
+        <h3 className="text-lg font-bold mb-4">Success Gallery</h3>
+        <button className="text-sm font-medium text-blue-600 hover:underline">View All</button>
+      </div>
+      <div className="grid grid-cols-1 sm:grid-cols-3 gap-4">
+        {items.map((item, index) => (
+          <div
+            key={index}
+            className="w-full h-40 bg-gray-100 rounded-lg overflow-hidden shadow-sm cursor-pointer"
+            onClick={() => onMediaClick?.(index)} 
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)